pkg/storagev2/batch.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package storage
+
+import "context"
+
+type Batch interface {
+	// We will only need to batch write operations
+	Put(Item) error
+	Delete(Key) error
+
+	Commit() error
+}
+
+type BatchedStore interface {
+	Batch(context.Context) (Batch, error)
```

Comment:
it seems that for current implementation there's no case in which creation of batch returns error, should we change the signature or should we keep it like this for future cases @aloknerurkar ?